build-logic/publishing/src/main/kotlin/build-logic.published-java-platform.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
+    id("java-platform")
```

Comment:
do you know if this plugin is the cause that running `./gradlew build` includes creating the javadoc?

Reply:
I do not think so. `java-platform` should be something that configures the project to serve as `pom.xml` (== type=pom, dependency-only project which can be used as a set of version constraints).

What do you mean by "creating javadocs"?

Reply:
I recently migrated some of Apache JMeter javadocs to Dokka-generated javadocs, and here's the pattern I arrived to: https://github.com/apache/jmeter/blob/88423edfb1a354a6170ca608d3fb074bb5b7f2ff/build-logic/jvm/src/main/kotlin/build-logic.dokka-javadoc.gradle.kts#L30-L47

It might help if you want "remove javadoc-generated javadocs" (see altering default javadoc classifier and removing it from `JAVADOC_ELEMENTS_CONFIGURATION_NAME` configuration artifacts).

Reply:
suddenly the build task depends on the javadoc task. I'll figure out why